unreleased_history/new_features/wait_for_compact.md

```diff
@@ -0,0 +1 @@
+Add `WaitForCompact()` to wait for all flush and compactions jobs to finish. Jobs to wait include the unscheduled (queued, but not scheduled yet).
```

Comment:
I guess `*` is missing on purpose to test the case that missing`*` will be automatically added.

Reply:
Yes :)